src/domains/transaction/pages/SendVote/SendVote.tsx

```diff
@@ -498,7 +508,8 @@ export const SendVote = () => {
 									onClose={() => navigate(`/profiles/${activeProfile.id()}/dashboard`)}
 									isBackDisabled={isSubmitting}
 									onBack={() => {
-										setActiveTab(Step.FormStep);
+										// Use the improved handleBack logic
+										handleBack();
```

Comment:
did this change in itself not result in the expected change? with other forms we don't explicitly track the error page either, so I wonder why we need a different approach here 🤔